docstring/src/test/java/io/cucumber/docstring/DocStringTypeRegistryDocStringConverterTest.java

```diff
@@ -16,49 +19,47 @@
         registry);
 
     @Test
-    void uses_doc_string_type_when_available() {
-        registry.defineDocStringType(new DocStringType(
-            JsonNode.class,
-            "json",
-            (String s) -> new ObjectMapper().readTree(s)));
-
+    void target_type_to_string_is_predefined() {
         DocString docString = DocString.create(
-            "{\"hello\":\"world\"}",
-            "json");
+            "hello world");
 
-        JsonNode converted = converter.convert(docString, Object.class);
-        assertThat(converted.get("hello").textValue(), is("world"));
+        String converted = converter.convert(docString, String.class);
+        assertThat(converted, is("hello world"));
     }
 
     @Test
-    void uses_target_type_when_available() {
-        registry.defineDocStringType(new DocStringType(
-            JsonNode.class,
-            "json",
-            (String s) -> new ObjectMapper().readTree(s)));
-
+    void converts_doc_string_to_doc_string() {
```

Comment:
It looks like some methods got shuffled around in this file. This makes reviewing rather difficult. Can you put them back in the same order as much as possible?

Reply:
Re-arranged them, to be in the same order as originally, though the first test from before has to be changed to throw an exception given that the functionality no longer works the same.

Reply:
Appreciated!